client/src/app/components/manage-inventory-items/manage-inventory-items.component.scss

```diff
@@ -43,3 +45,6 @@ mat-panel-title {
   justify-content: flex-end;
 }
 
+td , th {
+    text-align:center;
+}
```

Comment:
I noticed that only the table data text is being centered and not the table header. Is there a way to make them consistent?  

![image](https://user-images.githubusercontent.com/43306217/113651831-bbfed900-9660-11eb-9c46-cfd5551a6160.png)

Reply:
I addressed this issue in my latest commit.